README.md

```diff
@@ -85,20 +109,20 @@ https://github.com/jvkumar/delivery-finder/blob/master/assets/wholesfood2_1.png)
 *IMPORTANT* - Make sure you don't close the retailer's checkout page (from where you collected the cookie) ever while this finder is running. So let the page remain opened.
 
 ## Step 5
-Go to `covid19` folder and then subfolder `delivery-finder` subfolder where this finder codebase is residing. 
+Go to `covid19` folder and then subfolder `delivery-finder` subfolder where this finder codebase is residing.
 
 ## Step 6
```

Comment:
@parikhv15 since you are moving cookie configuration in `config/auth.js ` so step 6 would need change as well.